### ISSUE TITLE
Passing --print-format argument results in yay exiting with 'target-not-found' error

### ISSUE THREAD

**Issue by codeinred** [Jguer/yay#1744]
### Affected Version

```
yay v11.1.2 - libalpm v13.0.1
```

### Describe the bug

When `--print-format` is passed to yay when installing a package, yay will synchronize package databases before exiting with a 'target not found' error
![image](https://user-images.githubusercontent.com/36810712/165023327-1dda6ce6-3f96-4057-ad4b-87922776bb40.png)

### Reproduction Steps

1. Find a AUR package (I selected [scc](https://aur.archlinux.org/packages/scc))
2. Invoke yay with the package (eg, `yay --print-format "%n %v" -Syu scc`

### Expected behavior

yay should ignore the `--print-format` argument and install the packages as necessary. It doesn't actually have to process it, or apply it, just ignore it (and whatever operand is provided to it)

The reason this is relevant is because the ansible pacman module always passes `--print-format` by default, and as far as I can tell, there's no way to disable this. 

### Output

```json
{
	"aururl": "https://aur.archlinux.org",
	"buildDir": "/home/alecto/.cache/yay",
	"editor": "",
	"editorflags": "",
	"makepkgbin": "makepkg",
	"makepkgconf": "",
	"pacmanbin": "pacman",
	"pacmanconf": "/etc/pacman.conf",
	"redownload": "no",
	"rebuild": "no",
	"answerclean": "",
	"answerdiff": "",
	"answeredit": "",
	"answerupgrade": "",
	"gitbin": "git",
	"gpgbin": "gpg",
	"gpgflags": "",
	"mflags": "",
	"sortby": "votes",
	"searchby": "name-desc",
	"gitflags": "",
	"removemake": "ask",
	"sudobin": "sudo",
	"sudoflags": "",
	"requestsplitn": 150,
	"completionrefreshtime": 7,
	"bottomup": true,
	"sudoloop": false,
	"timeupdate": false,
	"devel": false,
	"cleanAfter": false,
	"provides": true,
	"pgpfetch": true,
	"upgrademenu": true,
	"cleanmenu": true,
	"diffmenu": true,
	"editmenu": false,
	"combinedupgrade": false,
	"useask": false,
	"batchinstall": false,
	"singlelineresults": false,
	"version": "11.1.2"
}
```

**Comment by Minei3oat** [comment]
No, the expected behavior is, that yay lists all packages matching the given arguments and exits. So simply ignoring the argument will lead to unexpected behavior, since requests with `--print-format` will never change things in pacman.

yay failes here, as it seems to forward the request to pacman, which won't find AUR packages:
https://github.com/Jguer/yay/blob/0fdfe79943112e7e7b72dee7b7eeb575a768681c/cmd.go#L336-L338

P.S.: community.general.pacman first creates an inventory with all installed/available packages, their versions and install reason for checking if changes are needed (with `--print-format`). Then all given packages are resolved (if not present in inventory, they are resolved with `-S --print-format`, so installed when `--print-format` is ignored, even when they should be removed/not updated). Later `-S --print-format` is used for building an install diff (Ignoring would lead to changes, even in `check_mode`). So just ignoring `--print-format` would break the module in multiple ways, but may solve your concrete issue.

**Comment by assapir** [comment]
Hey @Jguer - any updates regarding this? shouldn't it just pass the `--print-format` to pacman?

**Comment by Jguer** [comment]
@assapir this would require the option to be forwarded to `install` and install to do the gathering target process, print desired the format and then exit.
 
It should be doable but it's not as simple as just passing the argument (which is what's happening now which removes AUR support from it)

**Comment by ByJacob** [comment]
Any update on this topic?

**Comment by pythops** [comment]
this is blocking. I can't install AUR packages using ansible. Anyone has a suggestion for a workaround ?

**Comment by ByJacob** [comment]
```yaml
- name: Check install packages
  ansible.builtin.command: pacman -Qm
  register: installed_packages

- name: "Install package"
  become: false
  ansible.builtin.command: "yay --noconfirm --noprogressbar --needed --sync {{ item }}"
  when: item not in installed_packages.stdout 
  with_items:
    - d2-bin
```

Only these must be packages with AUR only, as the condition will not work for packages that you can normally pacman install [Ref](https://archlinux.org/pacman/pacman.8.html#_query_options_apply_to_em_q_em_a_id_qo_a).

**Comment by pythops** [comment]
Thanks, Yeah, there is always the `command` or `shell` plugin to use but I'm wondering if there is another workaround using `pacman` plugin
